src/support/mtx_rw.c

```diff
@@ -48,9 +48,8 @@
  *		uint8_t current;		// Current ticket
  *		uint8_t next;			// Next available ticket
  *		uint8_t reader;			// Read queue ticket
- *		uint8_t __notused;		// Padding
- *		uint16_t readers_active;	// Count of active readers
- *		uint16_t readers_queued;	// Count of queued readers
+ *		uint8_t readers_queued;		// Count of queued readers
```

Comment:
With this change, I think the comment [here](https://github.com/wiredtiger/wiredtiger/blob/develop/src/support/mtx_rw.c#L209) should be brought up to the general description block - I presume that's the reason why it's OK to use a smaller number for queued readers?

Reply:
Agreed, that comment is the reason this change is possible.  I'll update with your suggestion.